.gitleaks.toml

```diff
@@ -0,0 +1,5 @@
+[allowlist]
+  description = "Our test install exports a test only MINIO ACCESS KEY"
+  paths = [
+    ".github/workflows/scripts/install.sh",
```

Comment:
Is it worth changing the path to the `.github/workflows/scripts/` instead?

Reply:
The more precise the better.